### DISCUSSION TITLE
Using V3 login authentication for the first time takes more time than the subsequent ones.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [acheong08/ChatGPT#993] [General]
I noticed that the first time to use [V3 login authentication](https://github.com/acheong08/ChatGPT#command-line-1) takes more time than the subsequent login authentication with this method. Any tips/comments/explanations for this behavior will be appreciated.

Regards,
Zhao

**Comment by acheong08** [comment]
`tiktoken` cache
